skyportal/models.py

```diff
@@ -1635,17 +1645,23 @@ class FollowupRequest(Base):
     payload = sa.Column(
         psql.JSONB, nullable=True, doc="Content of the followup request."
     )
-    status = sa.Column(sa.String(), nullable=False, default="pending submission")
+    status = sa.Column(
```

Comment:
Is there a pre-determined set of valid statuses? If so, perhaps this should be an enum

Reply:
there is not.